src/python/pants/init/engine_initializer.py

```diff
@@ -231,13 +239,10 @@ def run_goal_rules(self, options_bootstrapper, options, goals, target_roots):
 
     return PANTS_SUCCEEDED_EXIT_CODE
 
-  def create_build_graph(self, target_roots, build_root=None):
-    """Construct and return a `BuildGraph` given a set of input specs.
-
-    :param TargetRoots target_roots: The targets root of the request.
-    :param string build_root: The build root.
-    :returns: A tuple of (BuildGraph, AddressMapper).
-    """
+  def create_build_graph(
+    self, target_roots: TargetRoots, build_root: Optional[str] = None,
```

Comment:
```suggestion
    self, 
    target_roots: TargetRoots, 
    build_root: Optional[str] = None,
```

Reply:
Going to stick with the original because this currently fits within 100 columns and this is how Black would format this. (I'm very eager for Black so that this is automatic)